sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveUDFSuite.scala

```diff
@@ -552,6 +552,16 @@ class HiveUDFSuite extends QueryTest with TestHiveSingleton with SQLTestUtils {
     }
   }
 
+  test("Invoke a persist hive function with call_function") {
+    val testData = spark.range(5).repartition(1)
+    withUserDefinedFunction("`default.custom_func`" -> true) {
```

Comment:
this is not a persist function. Can you check other tests in this file? we need to use `CREATE FUNCTION` to create persist functions.

Reply:
Got it.